app/forms/fields.py

```diff
@@ -219,7 +219,7 @@ def get_schema_defined_limit(answer_id, definition, answer_store):
             source_answer_id = definition.get('answer_id')
             answer_list = answer_store.filter(answer_id=source_answer_id)
             value = answer_list[0].get('value')
-            if not isinstance(value, int) and not isinstance(value, Decimal):
+            if not isinstance(value, int) and not isinstance(value, Decimal) and not isinstance(value, float):
```

Comment:
How are these float values now getting into the answer store? We should only have numbers of type integer or decimal.